packages/boilerplate/theme/nuxt.config.js

```diff
@@ -10,20 +10,20 @@ export default {
     title: 'Vue Storefront',
     meta: [
       { charset: 'utf-8' },
-      { name: 'viewport',
-        content: 'width=device-width, initial-scale=1' },
-      { hid: 'description',
-        name: 'description',
-        content: process.env.npm_package_description || '' }
+      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
+      { hid: 'description', name: 'description', content: process.env.npm_package_description || '' }
     ],
     link: [
       { rel: 'icon',
         type: 'image/x-icon',
         href: '/favicon.ico'
-      }, {
-        rel: 'preconnect',
-        href: 'https://fonts.googleapis.com'
-      }, {
+      },
+      {
+        rel: 'preload',
+        href: 'https://fonts.googleapis.com/css?family=Raleway:300,400,400i,500,600,700|Roboto:300,300i,400,400i,500,700&display=swap',
+        as: 'style'
+      },
+      {
         rel: 'preconnect',
         href: 'https://fonts.gstatic.com'
       }
```

Comment:
Maybe this should be part of `packages/core/nuxt-module/lib/modules/storefront-ui.js` plugin? 🤔

Or maybe fonts shouldn't be part of SFUI, but instead be provided to it by some CSS variable?
@filrak @andrzejewsky @justyna-13 @AdamPawlinski

Reply:
I think it shouldn't because you don't always use these fonts ;)